### ISSUE TITLE
«ImportError: cannot import name 'suppress_warnings' from 'numpy.testing'»

### ISSUE THREAD

**Issue by belonesox** [Nuitka/Nuitka#1199]
factory branch on commit  2670c2d1c8250618534e42f11d2260ec1108d663
compiling trivial file
```
import scipy
from scipy import stats

print(scipy.__version__)
```
results 
```
ImportError: cannot import name 'suppress_warnings' from 'numpy.testing' (/home/stas/projects/nuitka-wtf/nuitka-supress-warnings/tmp/ta/test-nuitka
.dist/numpy/testing/__init__.py)
```

Look pipenv-based reproduction
* Unpack  
[nuitka-supress-warnings.zip](https://github.com/Nuitka/Nuitka/files/7101012/nuitka-supress-warnings.zip)
* ``build-me-first.sh`` 
* ``show-the-bug.sh``

**Comment by kayhayen** [comment]
Having a fixed revision from git in the pipfile is I think not how it's supposed to be used, does pipenv really not allow you to put a branch there? Just curious, because naturally this normally would be the best way to know things from the lock file.

**Comment by kayhayen** [comment]
Managed to reproduce it. It seems the whole constraints thing from scipy, could be subjected to elimination there, but I won't go there. Surviving the `numpy.testing` removal is a first goal.

**Comment by kayhayen** [comment]
This is now solved on the factory branch.

**Comment by kayhayen** [comment]
https://nuitka.net/doc/factory.html

**Comment by kayhayen** [comment]
This was released as part of 0.6.17.6 hotfix.
